pkg/didcomm/protocol/issuecredential/states_test.go

```diff
@@ -335,18 +335,27 @@ func TestRequestReceived_CanTransitionTo(t *testing.T) {
 }
 
 func TestRequestReceived_ExecuteInbound(t *testing.T) {
-	followup, action, err := (&requestReceived{}).ExecuteInbound(&metaData{})
-	require.NoError(t, err)
-	require.Equal(t, &credentialIssued{}, followup)
-	require.NotNil(t, action)
+	t.Run("Successes", func(t *testing.T) {
+		followup, action, err := (&requestReceived{}).ExecuteInbound(&metaData{issueCredential: &IssueCredential{}})
+		require.NoError(t, err)
+		require.Equal(t, &credentialIssued{}, followup)
+		require.NotNil(t, action)
```

Comment:
NotNil or NotEmpty?

Reply:
Actually, it does not matter. `action` is a function. Which means that it can be only `nil` or a function.